modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/enricher/ValueProvidersParameterDeclarationEnricher.java

```diff
@@ -155,9 +162,9 @@ private void enrichParameter(OfValues resolverClass,
     paramDeclaration.addModelProperty(propertyBuilder.build());
 
     valueProviderModelConsumer
-        .accept(new ValueProviderModel(getRequiredParametersAliases(resolverParameters, containerParameterNames),
```

Comment:
I think we should add tests that assert the value provider model other than the check of the test extensions extension model.

Reply:
Where would you create those tests?

Reply:
I would create a test for the enricher where the extension declaration is mocked, and then enriched, then assert the enriched declaration.

Reply:
Tests added